ecl/hqlcpp/hqlnlp.cpp

```diff
@@ -709,15 +709,16 @@ ABoundActivity * HqlCppTranslator::doBuildActivityParse(BuildCtx & ctx, IHqlExpr
         else if (matchesConstantString(algorithmHint, "heap", true))
             algorithm = NLPAregexHeap;
 
-        nlpParse = createRegexContext(expr, code->workunit, options, reporter, algorithm);
+        nlpParse = createRegexContext(expr, code->workunit, options, algorithm);
     }
 
     gatherExplicitMatched(expr);
     doBuildParseTransform(instance->startctx, expr);            // also gathers all the MATCHED() definitions.
     doBuildParseSearchText(instance->startctx, expr);
     doBuildParseValidators(instance->nestedctx, expr);
     doBuildParseExtra(instance->startctx, expr);
-    noteFinishedTiming("compile:generate PARSE:prepare", startPrepareCycles);
+    if (options.timeTransforms)
+        noteFinishedTiming("compile:PARSE:prepare", startPrepareCycles);
```

Comment:
Because it corresponds to the PARSE keyword.  
I could change to lower case, but then it might be confused with the eclcc parser, so I would keep it as is, even though it isn't consistent.